import-export-cli/integration/apim/apim.go

```diff
@@ -2295,3 +2298,186 @@ func getTokenURL(host string, offset int) string {
 	port := 9443 + offset
 	return "https://" + host + ":" + strconv.Itoa(port) + "/oauth2/token"
 }
+
+// GetAPIPolicies : Get API Policies list of all types from APIM
+func (instance *Client) GetAPIPolicies(t *testing.T, offset, limit string) *APIPoliciesList {
+	var policyListResponse *APIPoliciesList
+
+	getPoliciesURL := instance.publisherRestURL + operationPolicyResourcePath
+
+	queryParams := "offset=" + offset + "&limit=" + limit
+
+	if offset != "-1" && limit != "-1" {
+		getPoliciesURL += "?" + queryParams
+	}
+
+	request := base.CreateGet(getPoliciesURL)
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.GetAPIPolicies()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	base.ValidateAndLogResponse("apim.GetAPIPolicies()", response, 200)
+
+	err := json.NewDecoder(response.Body).Decode(&policyListResponse)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	return policyListResponse
+}
+
+// DeleteAPIPolicy : Deletes API Policy from APIM using ID
+func (instance *Client) DeleteAPIPolicy(policyID, baseTest string) {
```

Comment:
As per the discussion, we do not need baseTest argument anymore.

Reply:
I think it's better to keep this. If someone wants to validate the response of the delete operation, this will be needed